geopandas/tests/util.py

```diff
@@ -69,10 +69,10 @@ def connect(dbname, user=None, password=None, host=None, port=None):
 
 def get_srid(df):
     """Return srid from `df.crs`."""
-    crs = df.crs
-    return (
-        int(crs["init"][5:]) if "init" in crs and crs["init"].startswith("epsg:") else 0
-    )
+    try:
+        return df.crs.to_epsg() or 0
+    except AttributeError:
```

Comment:
When does this return an AttributeError ?

Reply:
When it is None

Reply:
I think changing the code to make that more clear is a good idea.